infra/src/test/kotlin/fi/fta/geoviite/infra/projektivelho/FakeProjektiVelho.kt

```diff
@@ -32,7 +33,12 @@ class FakeProjektiVelho(port: Int, val jsonMapper: ObjectMapper): AutoCloseable
 
     fun search() {
         post(XML_FILE_SEARCH_PATH).respond(okJsonSerialized(
-            PVApiSearchStatus("", PVId("123"), Instant.now().minusSeconds(5), 3600)
+            PVApiSearchStatus(
+                PVDictionaryCode("asdf"),
```

Comment:
Nyt kun näitä täällä katsoo niin on tämä tosiaan tyyppinä vähän hassu 🤔 Vaikka näitä statuksia ei varsinaisesti dokumentoitu olekaan, niin kannattaisikohan tälle silti tehdä jonkinlainen sanitoitu tyyppi joka nimeäisi tämän vähän paremmin?

Reply:
joo katson tuota vielä